Blaise-Shyaka/Politico
# [#171591958] Create a reset password page

# What does this PR do?

It creates a page that allows users who have forgotten their passwords to create a new one.

# Description of tasks to be completed

- Create two pages
- One page will be used to collect the user's email, authenticate, and use that email to send a reset password link
- The second page will be used to create and confirm the new password
- Make the UI of both pages simple and responsive

# How should this be manually tested?

- Clone this repository and `cd /templates/html/`
- Open `auth-reset password.html` file
- Verify if it has a field to fill in an email 
- Resize your browser to verify its responsiveness
- Open `reset-password.html` file
- Verify if it has two fields to create and confirm the new password.
- Resize your browser to verify its responsiveness

# What are the relevant pivotal tracker stories?

#171591958

# Screenshots

![Screenshot (26)](https://user-images.githubusercontent.com/52489421/75888400-fda4cb00-5e33-11ea-8bf8-8ddbdeada010.png)
![Screenshot (27)](https://user-images.githubusercontent.com/52489421/75888412-009fbb80-5e34-11ea-8381-a1a0362e21a2.png)
![Screenshot (28)](https://user-images.githubusercontent.com/52489421/75888416-01385200-5e34-11ea-8292-295e308003dd.png)
![Screenshot (29)](https://user-images.githubusercontent.com/52489421/75888421-02697f00-5e34-11ea-8997-6826dfdfda10.png)
![Screenshot (24)](https://user-images.githubusercontent.com/52489421/75888423-039aac00-5e34-11ea-80fd-ec34684fba8a.png)
![Screenshot (25)](https://user-images.githubusercontent.com/52489421/75888426-04cbd900-5e34-11ea-9c85-d6a35878091a.png)
